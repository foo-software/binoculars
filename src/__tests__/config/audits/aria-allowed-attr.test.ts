import auditAriaAllowedAttr from '../../../config/audits/aria-allowed-attr';

describe('aria-allowed-attr audit', () => {
  it('should match snapshot', () => {
    expect(auditAriaAllowedAttr('en')).toMatchSnapshot();
  });
});
