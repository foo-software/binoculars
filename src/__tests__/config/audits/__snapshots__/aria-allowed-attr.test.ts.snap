// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aria-allowed-attr audit should match snapshot 1`] = `
Object {
  "description": "Each ARIA \`role\` supports a specific subset of \`aria-*\` attributes. Mismatching these invalidates the \`aria-*\` attributes. [Learn more](https://web.dev/aria-allowed-attr/).",
  "title": "\`[aria-*]\` attributes match their roles",
}
`;
